app/presenters/dc_time_presenter.rb

```diff
@@ -26,9 +26,11 @@ def convert
     time.in_time_zone(time_zone)
   end
 
-  def convert_and_format(format = FORMAT)
+  def convert_and_format(format = FORMAT, timezone_label: true)
```

Comment:
I think time zone is pretty helpful in admin. otherwise not sure if it is UTC or ET